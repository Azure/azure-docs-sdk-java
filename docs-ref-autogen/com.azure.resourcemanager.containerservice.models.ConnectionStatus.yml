### YamlMime:JavaType
uid: "com.azure.resourcemanager.containerservice.models.ConnectionStatus"
fullName: "com.azure.resourcemanager.containerservice.models.ConnectionStatus"
name: "ConnectionStatus"
nameWithType: "ConnectionStatus"
summary: "Defines values for ConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class ConnectionStatus extends ExpandableStringEnum<ConnectionStatus>"
constructors:
- "com.azure.resourcemanager.containerservice.models.ConnectionStatus.ConnectionStatus()"
fields:
- "com.azure.resourcemanager.containerservice.models.ConnectionStatus.APPROVED"
- "com.azure.resourcemanager.containerservice.models.ConnectionStatus.DISCONNECTED"
- "com.azure.resourcemanager.containerservice.models.ConnectionStatus.PENDING"
- "com.azure.resourcemanager.containerservice.models.ConnectionStatus.REJECTED"
methods:
- "com.azure.resourcemanager.containerservice.models.ConnectionStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.containerservice.models.ConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.containerservice.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-containerservice:2.1.0
