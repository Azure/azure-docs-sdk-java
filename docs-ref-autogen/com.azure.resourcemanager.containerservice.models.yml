### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.containerservice.models"
fullName: "com.azure.resourcemanager.containerservice.models"
name: "com.azure.resourcemanager.containerservice.models"
summary: "Package containing the data models for ContainerServiceManagementClient. Container Service Client."
classes:
- "com.azure.resourcemanager.containerservice.models.AgentPoolAvailableVersionsPropertiesAgentPoolVersionsItem"
- "com.azure.resourcemanager.containerservice.models.AgentPoolListResult"
- "com.azure.resourcemanager.containerservice.models.AgentPoolMode"
- "com.azure.resourcemanager.containerservice.models.AgentPoolType"
- "com.azure.resourcemanager.containerservice.models.AgentPoolUpgradeProfilePropertiesUpgradesItem"
- "com.azure.resourcemanager.containerservice.models.AgentPoolUpgradeSettings"
- "com.azure.resourcemanager.containerservice.models.Components1Q1Og48SchemasManagedclusterAllof1"
- "com.azure.resourcemanager.containerservice.models.ComponentsQit0EtSchemasManagedclusterpropertiesPropertiesIdentityprofileAdditionalproperties"
- "com.azure.resourcemanager.containerservice.models.ConnectionStatus"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceAgentPoolProfile"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceCustomProfile"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceDiagnosticsProfile"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceLinuxProfile"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceListResult"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceMasterProfile"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceNetworkProfile"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceOrchestratorProfile"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceOrchestratorTypes"
- "com.azure.resourcemanager.containerservice.models.ContainerServicePrincipalProfile"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceSshConfiguration"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceSshPublicKey"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceStorageProfileTypes"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceVMDiagnostics"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceVMSizeTypes"
- "com.azure.resourcemanager.containerservice.models.ContainerServiceWindowsProfile"
- "com.azure.resourcemanager.containerservice.models.CredentialResult"
- "com.azure.resourcemanager.containerservice.models.KeyVaultSecretRef"
- "com.azure.resourcemanager.containerservice.models.LoadBalancerSku"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterAadProfile"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterAddonProfile"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterAddonProfileIdentity"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterAgentPoolProfile"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterAgentPoolProfileProperties"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterApiServerAccessProfile"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterIdentity"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterIdentityUserAssignedIdentities"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterListResult"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterLoadBalancerProfile"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterLoadBalancerProfileManagedOutboundIPs"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterLoadBalancerProfileOutboundIPs"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterLoadBalancerProfileOutboundIpPrefixes"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterPoolUpgradeProfile"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterPoolUpgradeProfileUpgradesItem"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterPropertiesAutoScalerProfile"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterServicePrincipalProfile"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterSku"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterSkuName"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterSkuTier"
- "com.azure.resourcemanager.containerservice.models.ManagedClusterWindowsProfile"
- "com.azure.resourcemanager.containerservice.models.NetworkMode"
- "com.azure.resourcemanager.containerservice.models.NetworkPlugin"
- "com.azure.resourcemanager.containerservice.models.NetworkPolicy"
- "com.azure.resourcemanager.containerservice.models.NetworkProfile"
- "com.azure.resourcemanager.containerservice.models.OSType"
- "com.azure.resourcemanager.containerservice.models.OpenShiftAgentPoolProfileRole"
- "com.azure.resourcemanager.containerservice.models.OpenShiftContainerServiceVMSize"
- "com.azure.resourcemanager.containerservice.models.OpenShiftManagedClusterAadIdentityProvider"
- "com.azure.resourcemanager.containerservice.models.OpenShiftManagedClusterAgentPoolProfile"
- "com.azure.resourcemanager.containerservice.models.OpenShiftManagedClusterAuthProfile"
- "com.azure.resourcemanager.containerservice.models.OpenShiftManagedClusterBaseIdentityProvider"
- "com.azure.resourcemanager.containerservice.models.OpenShiftManagedClusterIdentityProvider"
- "com.azure.resourcemanager.containerservice.models.OpenShiftManagedClusterListResult"
- "com.azure.resourcemanager.containerservice.models.OpenShiftManagedClusterMasterPoolProfile"
- "com.azure.resourcemanager.containerservice.models.OpenShiftRouterProfile"
- "com.azure.resourcemanager.containerservice.models.OperationListResult"
- "com.azure.resourcemanager.containerservice.models.OrchestratorProfile"
- "com.azure.resourcemanager.containerservice.models.OrchestratorVersionProfile"
- "com.azure.resourcemanager.containerservice.models.OutboundType"
- "com.azure.resourcemanager.containerservice.models.PrivateEndpoint"
- "com.azure.resourcemanager.containerservice.models.PrivateEndpointConnectionProvisioningState"
- "com.azure.resourcemanager.containerservice.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.containerservice.models.PurchasePlan"
- "com.azure.resourcemanager.containerservice.models.ResourceReference"
- "com.azure.resourcemanager.containerservice.models.ScaleSetEvictionPolicy"
- "com.azure.resourcemanager.containerservice.models.ScaleSetPriority"
- "com.azure.resourcemanager.containerservice.models.TagsObject"
- "com.azure.resourcemanager.containerservice.models.UserAssignedIdentity"
enums:
- "com.azure.resourcemanager.containerservice.models.Count"
- "com.azure.resourcemanager.containerservice.models.ResourceIdentityType"
interfaces:
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.Definition"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.Blank"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.NetworkProfileDefinition"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.NetworkProfileDefinitionStages"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.NetworkProfileDefinitionStages.Blank"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.NetworkProfileDefinitionStages.WithAttach"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.NetworkProfileDefinitionStages.WithDnsServiceIP"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.NetworkProfileDefinitionStages.WithDockerBridgeCidr"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.NetworkProfileDefinitionStages.WithNetworkPolicy"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.NetworkProfileDefinitionStages.WithPodCidr"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.NetworkProfileDefinitionStages.WithServiceCidr"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.WithAddOnProfiles"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.WithAgentPool"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.WithDnsPrefix"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.WithGroup"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.WithLinuxRootUsername"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.WithLinuxSshKey"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.WithNetworkProfile"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.WithServicePrincipalClientId"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.WithServicePrincipalProfile"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.DefinitionStages.WithVersion"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.Update"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.UpdateStages"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.UpdateStages.WithAddOnProfiles"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.UpdateStages.WithAgentPool"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.UpdateStages.WithNetworkProfile"
- "com.azure.resourcemanager.containerservice.models.KubernetesCluster.UpdateStages.WithRBAC"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.Definition"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.DefinitionStages"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.DefinitionStages.Blank"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.DefinitionStages.WithAgentPoolMode"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.DefinitionStages.WithAgentPoolType"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.DefinitionStages.WithAgentPoolVirtualMachineCount"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.DefinitionStages.WithAttach"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.DefinitionStages.WithMaxPodsCount"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.DefinitionStages.WithOSDiskSize"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.DefinitionStages.WithOSType"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.DefinitionStages.WithVirtualNetwork"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.Update"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.UpdateStages"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.UpdateStages.WithAgentPoolMode"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterAgentPool.UpdateStages.WithAgentPoolVirtualMachineCount"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusterUpgradeProfile"
- "com.azure.resourcemanager.containerservice.models.KubernetesClusters"
metadata: {}
package: "com.azure.resourcemanager.containerservice.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-containerservice:2.1.0
