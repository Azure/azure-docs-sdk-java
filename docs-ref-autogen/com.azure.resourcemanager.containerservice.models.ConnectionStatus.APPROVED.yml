### YamlMime:JavaMember
uid: "com.azure.resourcemanager.containerservice.models.ConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.containerservice.models.ConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "ConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.containerservice.models.ConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.containerservice.models.ConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "ConnectionStatus.APPROVED"
  summary: "Static value Approved for ConnectionStatus."
  syntax: "public static final ConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.containerservice.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-containerservice:2.1.0
